clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/processor/CloudLoggingProcessor.java

```diff
@@ -71,6 +73,14 @@ public AuditLogRequest process(AuditLogRequest auditLogRequest) throws LogProces
                           new TypeReference<Map<String, ?>>() {})))
               .setLogName(getLogNameFromLogType(auditLogRequest.getType()))
               .setLabels(auditLogRequest.getLabelsMap())
+              .setOperation(
+                  Operation.of(
+                      auditLogRequest.getOperation().getId(),
```

Comment:
This is weird. Both https://github.com/abcxyz/lumberjack/blob/563a9dd75e25d3645e6bebecc506d7770679f17c/protos/v1alpha1/audit_log_request.proto#L82 and https://github.com/googleapis/googleapis/blob/master/google/logging/v2/log_entry.proto#L154 use `LogEntryOperation`. But in Java, `setOperation` expects an `Operation` instead?

Reply:
Yes, the LogEntry here is different then the log_entry proto, it has a internal/package level method to convert log entry proto.

Reply:
Not the cleanest thing maybe, but both for Operation and for Timestamp, after our review round yesterday, given those methods seem slightly more generalized than what we are doing here, I thought maybe we should verbatim copy/paste those two internal methods and use them as helpers instead of manually doing the conversion inline?

Reply:
Thanks for calling this out! I quickly checked the repo and didn't see other places that need the conversion helpers, and probably unlikely in future as well since we only need this LogEntry to write to cloud logging for java. We can do the refactor when needed, what do you think?

Reply:
Sure, either way fine by me.